examples/chart/teleport-cluster/values.schema.json

```diff
@@ -49,7 +49,39 @@
         },
         "proxy": {
             "$id": "#/properties/proxy",
-            "type": "object"
+            "type": "object",
+            "properties": {
+                "teleportConfig": {
+                    "type": "object"
+                },
```

Comment:
```suggestion
```
I don't think this is required for the PR and may cause unrelated issues - probably best to remove it. We don't have the same value under `auth`.

Reply:
I saw `teleportConfig` exists on the current values, thats why I added it on [values.schema.json](https://github.com/gravitational/teleport/pull/39290/files/206351dfbc134bcfaf2eb214f38a673b5dafa251#diff-d3656ccc8503ae6ef6717561811a6cead6654b8b2f5d79692b6cc68ebf982f9b), we can remove it as you suggest.